::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 23 Nov 2017 03:33:01 +0200
Subject: [PATCH] maas: region: Allow subnets without fabrics

Configuring subnets should be possible without explicitly configuring
fabrics.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---

diff --git a/maas/region.sls b/maas/region.sls
--- a/maas/region.sls
+++ b/maas/region.sls
@@ -186,7 +186,9 @@
   - name: maas.process_subnets
   - require:
     - module: maas_config
+    {%- if region.get('fabrics', False)  %}
     - module: maas_fabrics
+    {%- endif %}
 {%- endif %}

 {%- if region.get('devices', False)  %}
